README.md

```diff
@@ -1,5 +1,5 @@
-[![Build Status](https://travis-ci.org/coveo/fmt-maven-plugin.svg?branch=master)](https://travis-ci.org/coveo/fmt-maven-plugin)
-[![license](http://img.shields.io/badge/license-MIT-brightgreen.svg)](https://github.com/coveo/fmt-maven-plugin/blob/master/LICENSE)
+[![Build Status](https://travis-ci.org/coveooss/fmt-maven-plugin.svg?branch=master)](https://travis-ci.org/coveooss/fmt-maven-plugin)
```

Comment:
Hmm, following the work in #96 I suppose this badge should be [replaced](https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge) altogether.

Reply:
K, added one more commit for that.